hawkular-inventory-bus-api/src/main/java/org/hawkular/inventory/bus/api/DataEntityEvent.java

```diff
@@ -29,8 +30,8 @@
     public DataEntityEvent() {
     }
 
-    public DataEntityEvent(Action.Enumerated action, DataEntity object) {
-        super(action, object);
+    public DataEntityEvent(Action.Enumerated action, Tenant tenant, DataEntity object) {
```

Comment:
We don't want to expose events belonging to a different tenant (security issues)

Right, it's not used. I was assuming that `inventory.observable(interest)` would provide an `Observable<? extends InventoryEvent>` so that I could filter the events of a given tenants but this was not the case.

I guess the InventoryEvent is used only for the bus communication. However, it may be convenient to have the tenant id in the json payload, instead of parsing the cpaths, wdyt?

At the end of the day I used this http://git.io/vuzrI and http://git.io/vuzrs for filtering by tenant id.

I don't like how the `RestEvents.getFilter()` method looks like, because if a new action is added, this place needs to be updated.. Other option would be to have these filters (String tenantId -> boolean) inside the `Action` itself.